app/models/user.rb

```diff
@@ -13,7 +13,7 @@ class User < RedisStore
 
   # vaafi attributes
   attribute :edipi
-  attribute :issue_instant
+  attribute :login_time
```

Comment:
I'm only familiar with using Devise for authentication and it has a `last_signed_in` column that it updates. I would assume the platform team could set a similar attribute after a user logs in and that it would be in UTC